docs/working.md

```diff
@@ -111,18 +111,43 @@ In the future, it's likely coreos-assembler will also support something
 like `overrides/src` which could be a directory of symlinks to local
 git repositories.
 
-## Using cosa run --bind-ro for even faster iteration
+## Using `cosa build-fast`
```

Comment:
Minor nit: it doesn't look like we use back ticks in the subheadings for the rest of that page.